doc/developer/design/20220226_multiwriter_persist.md

```diff
@@ -0,0 +1,205 @@
+## Overview
```

Comment:
It could be obvious to others, but I think it would be worthwhile to add a paragraph on the motivation why multi-writer is considered relevant.

Reply:
Sorry, it should be obvious only because we walked through the STORAGE design proposal I'm working on on Friday. The motivation for having multiwriter collections is to support out HA story where we'll have multiple independent clusters all computing the same views over definite collections and wanting to sink them back to STORAGE.

Having the persistent collection itself be multi-writer means that readers of the collection don't need to be aware of how many HA instances are there, if they crashed, restarted, etc.